advanced_functionality/multi_model_xgboost_home_value/xgboost_multi_model_endpoint_home_value.ipynb

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
<p>The title needs to be updated to XGBoost</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/935/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Nice catch, updated to XGBoost and also fixed BYO notebook title.